Solution1/Solution1.Tests/BL.Tests/Service/WeatherApiServiceTests.cs

```diff
@@ -0,0 +1,64 @@
+﻿using BusinessLayer.DTOs;
+using BusinessLayer.Service.Abstract;
+using KellermanSoftware.CompareNetObjects;
+using Moq;
+using Moq.Protected;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Json;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+
+
+namespace Solution1.Tests.BL.Tests
```

Comment:
folders aren't names with dots. Rename folder and namespace to BL. .Tests.BL.Tests why tests twice